packages/jest-cli/src/TestRunner.js

```diff
@@ -379,6 +379,10 @@ class TestRunner {
   _setupReporters() {
     const config = this._config;
 
+    if (config.reporters) {
+      return this._addCustomReporters(config.reporters);
```

Comment:
I have added the first iteration of this which used regardless of reporters option specified or not.